### DISCUSSION TITLE
Dashboard asks credentials (username/password)

### DISCUSSION THREAD

**Discussion by safwanadnan** [devlikeapro/waha#534] [Q&A]
I bought waha plus and set it up and the terminal logs show it is ready to use but once I go to the dashboard on the port that it is running it asks for some login credentials. I never configured any credentials and the one give for the waha key do not work, can anyone please help?

**Comment by devlikepro** [comment]
Hi! 
The default is `waha/waha` https://waha.devlike.pro/docs/how-to/waha-dashboard/#overview, sorry about the confusion

Consider setting up your username/password via docker compose file or docker run command
https://waha.devlike.pro/docs/how-to/install/

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by EduOsc-svg** [comment]
me too , it doesnt work even when i used docs https://waha.devlike.pro/docs/how-to/install/ , in context i used waha/waha:latest
